packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -367,11 +441,42 @@ contract Bridge is EssentialContract, IBridge {
         return _ctx;
     }
 
+    /// @notice Returns invocation delay values.
+    /// @dev Bridge contract deployed on L1 shall use a non-zero value for better
+    /// security.
+    /// @return invocationDelay The minimal delay in second before a message can be executed since
+    /// and the time it was received on the this chain.
+    /// @return invocationExtraelay The extra delay in second (to be added to invocationDelay) if
+    /// the transactor is not the preferredExecutor who acknowledged the reception of this message.
+    function getInvocationDelays()
+        public
+        view
+        virtual
+        returns (uint256 invocationDelay, uint256 invocationExtraelay)
+    {
+        invocationDelay = 0;
+        invocationExtraelay = 0;
```

Comment:
This code will use `block.chainid` here to select the correct values? If so, maybe the L1 values can still be defined as public constants in this smart contract, this function then just selects them when on L1, otherwise this function just returns zeros.

Reply:
Ah I see. I guess the benefit of the `block.chainid` is no different contracts needed, but indeed not much difference.